service/src/main/java/com/sedliarov/learningtable/service/GroupService.java

```diff
@@ -18,6 +18,8 @@ public interface GroupService {
 
   void deleteGroup(UUID id);
 
+  void deleteGroups();
```

Comment:
We don't use this in controller and my question, For what we have this method?

Reply:
Done